lib/runner.js

```diff
@@ -331,7 +331,7 @@ class Runner {
             try {
                 service = require(`wdio-${serviceName}-service`)
             } catch (e) {
-                if (!e.message.match(`Cannot find module '${serviceName}'`)) {
+                if (!e.message.match(`Cannot find module '${serviceName}'`).length) {
```

Comment:
Why length? If the message doesn't match it returns null, try `"foo".match(/bar/)`